### DISCUSSION TITLE
Installation of v1.16.0 .NETFramework 4

### DISCUSSION THREAD

**Discussion by MrFanciful** [DaveSkender/Stock.Indicators#541] [Q&A] [answered]
Hi

Starting off that I come from mostly a Python background but trade with cTrader that requires bots and indicators to be coded in C# that I'm fairly new to, so please excuse my newbie ignorance.

I'm trying to install v1.16.0 into VS2019 using NuGet. I get the following error 

> Could not install package 'Skender.Stock.Indicators 1.16.0'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.0,Profile=Client', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author. 

I've tried ignoring dependencies but still got the same error. Not sure if your package would have worked even if it had installed.

Am I just out of luck using your package with this framework?

**Comment by DaveSkender** [comment, marked as answer]
The oldest [framework we target](https://daveskender.github.io/Stock.Indicators/#frameworks-targeted) is .NET Framework 4.6.1 so you have a couple options to try:

1. [Update your C# project framework](https://docs.microsoft.com/en-us/visualstudio/ide/visual-studio-multi-targeting-overview) version to 4.6.1 or newer.
2. Fork this repo and add 4.0 targeting to build an older version of the DLL yourself; this would also require that you manually add the DLL to your project.  There's a chance the current source code will not support version 4.0 and will require modifications.
